web-ui/demo/src/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import ReactDOM from "react-dom";
+import {DragDropContext, Droppable, Draggable} from "react-beautiful-dnd"
+
+const getItems = count => Array.from({length: count}, (v, k) => k)
+    .map(k => ({
+        id: `item-${k}`,
+        content: `item${k}`
+    }));
+
+const reorder = (list, startIndex, endIndex) => {
+    const result = Array.from(list);
+    const[removed] = result.splice(startIndex, 1);
+    result.splice(endIndex, 0, removed);
+    return result;
+};
+
+const grid = 0;
+
+const getItemStyle = (isDragging, draggableStyle) => ({
+    userSelect: "none",
+    padding: grid*2,
+    margin: '0 0 ${grid}px 0',
+
+    background: isDragging ? "lightgreen" : "grey",
+
+    ...draggableStyle
+});
+
+const getListStyle = isDraggingOver => ({
+  background: isDraggingOver ? "lightblue" : "lightgrey",
+  padding: grid,
+  width: 250
+});
+
+class App extends Component {
```

Comment:
This probably isn't a big deal but the code to be more uniform, we may want to use functional components and hooks here